### DISCUSSION TITLE
Can't make links open in other apps

### DISCUSSION THREAD

**Discussion by bepvte** [InfinityLoop1308/PipePipe#481] [Q&A]
When PipePipe is installed, no matter how I configure the "open links in app" setting in both PipePipe and youtube, youtube links always open in pipepipe. I want to use it for only some videos, is there something I'm doing wrong?
![Screenshot_20241017_175421_Settings](https://github.com/user-attachments/assets/52c052a6-d25a-4661-a434-f5e287cc5093)

**Comment by InfinityLoop1308** [comment]
This behavior is controlled by your system, so you need to check your system settings - something like 'open by default'.
